csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DotNet.cs

```diff
@@ -143,7 +143,7 @@ private static BuildScript DownloadDotNet(IBuildActions actions, ILogger logger,
             // See https://docs.microsoft.com/en-us/dotnet/core/tools/global-json
             var versions = new List<string>();
 
-            foreach (var path in files.Where(p => p.EndsWith("global.json", StringComparison.Ordinal)))
+            foreach (var path in files.Where(p => p.EndsWith(Path.DirectorySeparatorChar + "global.json", StringComparison.Ordinal)))
```

Comment:
Did you run into a case where a file was name `myglobal.json` or something similar?
nit: Maybe consider using `Path.GetFileName`? (same comment for the other similar change)

Reply:
Thanks for suggesting `Path.GetFileName`, that handles two other cases too. I'll change it...

I was putting together an integration test, and temporarily renamed the `packages.config` to `x_packages.config`, and then I was surprised that the packages were still picked up by the dependency manager.